bin/build/src/release/version_info.clj

```diff
@@ -5,29 +5,34 @@
    [clj-http.client :as http]
    [metabuild-common.core :as u]
    [release.common :as c]
-   [release.common.github :as github]))
+   [release.common.github :as github])
+  (:import
+   (java.time LocalDate)))
 
-(defn- version-info-filename []
-  (case (c/edition)
+(set! *warn-on-reflection* true)
+
+(defn- version-info-filename
+  [edition]
+  (case edition
     :oss "version-info.json"
     :ee  "version-info-ee.json"
```

Comment:
maybe not the most elegant testing suggestion:

```suggestion
  (case edition
    :oss "version-info-test.json"
    :ee  "version-info-ee-test.json"
```

Reply:
oh yeah, won't actually work because `workflow_dispatch` actions only work on the master branch 😢